postgres/berries.py

```diff
@@ -53,9 +53,11 @@ def get_berries_blob(cur, berry_id=None, berry_name=None):
 
 def get_berries_api(berry_id_or_name):
     response = requests.get(f"https://pokeapi.co/api/v2/berry/{berry_id_or_name}")
-    blob = response.json()
-    return blob
-
+    if response:
+        blob = response.json()
+        return blob
+    else:
+        raise AttributeError("ID not in API")
```

Comment:
It's always tempting to have an else block anytime you have an if block.  But you should really always think if this else needed?  In this case no.

```python
if response:
    return response.json()
raise AttributeError("ID not in API")
 ```
 
 or even better would be a guard clause like
 ```python
 if not response:
    raise AttributeError("ID not in API")
return response.json()
```

Reply:
Also, I have a hard time believing the response is Falsy even if the ID or _Name_ isn't present.